.env

```diff
@@ -0,0 +1 @@
+REACT_APP_YOUTUBE_API_KEY=AIzaSyBtyHxZNAYDhPiKVWFfpMfSwsx7nDSHDEM
```

Comment:
it is great that you have an `.env` file but the idea behind this is to not upload the file to github.  Try adding this file to your `.gitignore` file